tests/test_functional.py

```diff
@@ -23,7 +23,8 @@
 import pytest
 import six
 
-from conftest import mock_connection_params
+import mock
```

Comment:
mocking shouldn't be done in this test module; it goes in the Connections module.  see comments below.